dbms/src/Storages/DeltaMerge/tests/gtest_dm_storage_delta_merge.cpp

```diff
@@ -767,6 +784,14 @@ try
         SelectQueryInfo query_info;
         query_info.query = std::make_shared<ASTSelectQuery>();
         query_info.mvcc_query_info = std::make_unique<MvccQueryInfo>(ctx->getSettingsRef().resolve_locks, std::numeric_limits<UInt64>::max(), scan_context);
+        const google::protobuf::RepeatedPtrField<tipb::Expr> pushed_down_filters{};
+        query_info.dag_query = std::make_unique<DAGQueryInfo>(
+            google::protobuf::RepeatedPtrField<tipb::Expr>(),
+            pushed_down_filters, // Not care now
+            std::vector<TiDB::ColumnInfo>{}, // Not care now
+            std::vector<int>{},
+            0,
+            ctx->getTimezoneInfo());
```

Comment:
seems can remove?

Reply:
It should be keep otherwise the UT could not pass